go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/openshift/build-machinery-go v0.0.0-20230306181456-d321ffa04533
 	github.com/openshift/client-go v0.0.0-20230503144108-75015d2347cb
 	github.com/openshift/cluster-policy-controller v0.0.0-20231017103018-8af8cf302764
-	github.com/openshift/library-go v0.0.0-20230908101047-d058c695f677
+	github.com/openshift/library-go v0.0.0-20231030114140-33085da695c8
```

Comment:
Can we delete the patch file in this commit, too? Maybe that's somewhere else in the PR and I missed it?

Reply:
Oh, never mind, I was mixing up this PR and #2528